src/test/java/org/opensearch/sdk/TestExtensionInterfaces.java

```diff
@@ -123,15 +129,6 @@ public void testGetFieldFilter() {
         Assertions.assertNotNull(predicate);
     }
 
-    @Test
```

Comment:
Looks like you have an older version of this class before this was merged.  So you shouldn't remove this.

Rebasing or merging to main should resolve this, or you can copy-paste this back from the current class on main.